java/build.gradle.kts

```diff
@@ -19,19 +19,21 @@ plugins {
 // The version is specified here to prevent an inconsistent version from being seen by different tasks.
 version = System.getenv("JAVA_PROTOS_VERSION") ?: "0.1.0-SNAPSHOT"
 
-val grpcProtobufVersion = "3.21.2"
-val grpcVersion = "1.47.0"
-val guavaVersion = "31.1-android"
+val grpcProtobufVersion = "3.22.2"
+val grpcVersion = "1.53.0"
 
 dependencies {
-    implementation("io.grpc:grpc-protobuf:$grpcVersion")
-    implementation("com.google.api.grpc:proto-google-common-protos:2.0.1")
+    api(platform("io.grpc:grpc-bom:$grpcVersion"))
```

Comment:
nice, thanks for finding the bom.

does this need to be an `api` dep though?

Reply:
Gradle uses both api and implementation in their documentation. I used api because some of the dependencies are api dependencies. I'll try again to find out what api vs implementation means for a bom.